config/samples/ccruntime/base/ccruntime.yaml

```diff
@@ -10,7 +10,7 @@ spec:
       node-role.kubernetes.io/worker: ""
   config:
     installType: bundle
-    payloadImage: quay.io/confidential-containers/runtime-payload:kata-containers-5b7009f2f9cdb39099b97a4f868b53ec9f3c383f
+    payloadImage: quay.io/confidential-containers/runtime-payload:kata-containers
```

Comment:
I have noticed the tag `kata-containers` does not exist even if it will be updated.